src/components/form/fields/ConnectedSelect.tsx

```diff
@@ -10,18 +10,25 @@ import { default as metaDataResolver, MetaDataProps } from '../../../services/me
 
 interface OwnProps extends SelectProps, CollectionEditViewProps {
     onChange: (value: string, property: Property) => void;
+    shownAsMultipleItems?: boolean;
 }
 
-export type FullProps = OwnProps & RouteComponentProps<{ dataSet: string }> & MetaDataProps;
+export type Props = OwnProps & RouteComponentProps<{ dataSet: string }> & MetaDataProps;
 
-const SelectField: SFC<FullProps> = ({ name, selected, metadata, onChange }) => {
+const SelectField: SFC<Props> = ({ name, selected, metadata, onChange, shownAsMultipleItems = false }) => {
     const collection: CollectionMetadata | null =
         metadata && metadata.dataSetMetadata && metadata.dataSetMetadata.collection
             ? metadata.dataSetMetadata.collection
             : null;
 
+    const hiddenOptions = ['rdf_type'];
```

Comment:
This kinda smells like [an anti pattern](https://en.wikipedia.org/wiki/Magic_string)...? (correct me if I'm wrong)
Is there no way better way to do this?